3D-microservices-observability/node-js-app/beachshirt/controllers/stylingController.js

```diff
@@ -0,0 +1,41 @@
+/**
+* Driver for styling service which manages different styles of shirts and takes orders for a shirts
+* of a given style.
+* @author Yogesh Prasad Kurmi (ykurmi@vmware.com)
+*/
+const request = require('request');
+const uuidv1 = require('uuid/v1');
+const utils = require('../utils');
+
+module.exports = (app, config, log) => {
+    app.get('/style', (req, res) => {
+        res.json(
+            [
+                {
+                    "name": "Wavefront",
+                    "url": "WavefrontURL"
+                },
+                {
+                    "name": "BeachOps",
+                    "url": "BeachOpsURL"
+                }
+            ]
+        );
+    });
+
+    app.get('/style/:id/make', (req, res) => {
+        if (Math.floor(Math.random() * 10) == 0) {
```

Comment:
the error generation code could be extracted for reuse.  Also, the message is misleading, since this isn't the order service